core/collateral/engine.go

```diff
@@ -2323,6 +2321,17 @@ func (e *Engine) getTransferFundsTransferRequest(ctx context.Context, t *types.T
 			// we always pay onto the pending transfers accounts
 			toAcc = e.GetPendingTransfersAccount(t.Amount.Asset)
 
+		case types.AccountTypeVestedRewards:
+			fromAcc = e.GetOrCreatePartyVestedRewardAccount(ctx, t.Owner, t.Amount.Asset)
+			if err != nil {
```

Comment:
where is err coming from? you're creating the account if it doesn't exist so I'm not sure what you're doing here with the check.

Reply:
yea bad copy pasta of the previous lines.